pandas/tests/resample/test_resampler_grouper.py

```diff
@@ -542,6 +542,66 @@ def test_resample_empty_Dataframe(keys):
     tm.assert_frame_equal(result, expected)
 
 
+def test_resample_no_columns():
+    # GH#52484
+    df = DataFrame(
+        index=Index(
+            pd.to_datetime(
+                ["2018-01-01 00:00:00", "2018-01-01 12:00:00", "2018-01-02 00:00:00"]
+            ),
+            name="date",
+        )
+    )
+    result = df.groupby([0, 0, 1]).resample(rule=pd.to_timedelta("06:00:00")).mean()
```

Comment:
Some very odd behavior here. When I specify the groupings via a list (`[0, 0, 1]`), this gets encoded as a 32-bit int in the index of the result only on 64-bit Windows.